src/com/sun/ts/tests/jaxws/common/xml/common.xml

```diff
@@ -926,7 +926,7 @@
                       </classpath>
                   </taskdef>
                   <wsgen 
-                     fork="true"
+                     fork="false"
```

Comment:
Would it be better if this is made configurable via ts.jte? Similar to `wsgen.debug` and `wsgen.verbose`.

Reply:
this looks wrong. It looks more like a bug in wsgen. Why are you trying to mask/workaround issues instead of trying to make sure they get fixed?

Reply:
@lukasj - For issue with wsgen, is that glassfish team or metro team, that we need to connect for resolution?

Reply:
tooling (wsgen/import) is taken from metro. In any case, I'm testing the fix already, so I think it will get to GF later today

Reply:
btw: generic advice is - if stacktrace contains `org.glassfish.webservices` it's for GF, otherwise for Metro (usually for packages `com.sun.xml.ws` or `com.sun.tools.ws`)